cfgov/unprocessed/apps/paying-for-college/css/college-costs/financial-item.less

```diff
@@ -112,11 +131,38 @@
       margin-top: unit(5px / @base-font-size-px, em);
     }
   }
+
+  .o-costs-group_content {
+    .u-clearfix();
+
+    &__transition {
+      transition: max-height @flyout__transition-speed ease-in-out;
+    }
+
+    &.u-is-animating {
+      overflow: hidden;
+    }
+  }
```

Comment:
Does it still work if you remove all this?

Reply:
Yep! 👍